runelite-client/src/main/java/net/runelite/client/plugins/reportbutton/ReportButtonPlugin.java

```diff
@@ -134,6 +135,16 @@ public void onGameTick(GameTick tick)
 		}
 	}
 
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (!event.getGroup().equals("reportButton"))
+		{
+			return;
+		}
+		updateTimeFormat();
```

Comment:
Since only "switchTimeFormat" is acted upon here, check both the group and key.
```suggestion
		if (event.getGroup().equals("reportButton") && event.getKey().equals("switchTimeFormat"))
		{
			updateTimeFormat();
		}
```